components/Vote/Vote.tsx

```diff
@@ -1,31 +1,41 @@
-import { useCallback, useState } from "react";
+import { Dispatch, SetStateAction, useCallback } from "react";
 import cx from "classnames";
-
+import React from "react";
+type voteProps = {
```

Comment:
The name of a type is usually PascalCase
```suggestion
type VoteProps = {
```